### ISSUE TITLE
Use https://www.npmjs.com/package/mesos

### ISSUE THREAD

**Issue by stelcheck** [Wizcorp/frontrunner#15]
https://www.npmjs.com/package/mesos provides the API to operate with the Marathon cluster, which would make the code of this project easier to understand I think @MiLk any thoughts?

**Comment by MiLk** [comment]
Should we really use a dependency for one API call?
https://github.com/silas/node-mesos#marathontaskscallback
https://github.com/Wizcorp/frontrunner/blob/develop/lib/marathon.js#L31-L48

Not sure it makes sense.

**Comment by stelcheck** [comment]
We could get rid of zookeeper and get events through that library it appears.

**Comment by MiLk** [comment]
With event subscriptions?
https://github.com/silas/node-mesos#marathoneventsubscriptionregisteropts-callback
https://mesosphere.github.io/marathon/docs/rest-api.html#event-subscriptions

**Comment by stelcheck** [comment]
Yes. Since Marathon appears to be providing all we need and them some, it seems to be it would be more natural to follow their ways.
